pkg/pmem-device-manager/pmd-util.go

```diff
@@ -59,7 +59,7 @@ func FlushDevice(dev PmemDeviceInfo, blocks uint64) error {
 	if blocks == 0 {
 		glog.V(5).Infof("Wiping entire device: %s", dev.Path)
 		// use one iteration instead of shred's default=3 for speed
-		if _, err := pmemexec.RunCommand("shred", "-n", "1", dev.Path); err != nil {
+		if _, err := pmemexec.RunCommandNoMutex("shred", "-n", "1", dev.Path); err != nil {
```

Comment:
A comment explaining why this doesn't use `RunCommand` would be worthwhile here.

Reply:
comment added, force pushed